cheap_repr/__init__.py

```diff
@@ -234,7 +256,7 @@ def __init__(self, level, func, target_length):
         self.func = func
         self.target_length = target_length
 
-    def repr_iterable(self, iterable, left, right, end=False, length=None):
+    def repr_iterable(self, iterable: abc.Sequence, left: str, right: str, end: bool = False, length: t.Optional[int] = None) -> str:
```

Comment:
`iterable` is an Iterable, not necessarily a Sequence. Ideally, there would be overloads ensuring that `length` is provided if `__len__` doesn't exist, but it's not important.